### PR TITLE
Add support for finalfrontier embeddings.

### PR DESCRIPTION
**Pull request by danieldk**

### CODE REVIEW

File: toponn/src/input.rs [hunk 1]
```diff
@@ -1,18 +1,51 @@
...
+        };
+
+        embed
+    }
+}
+
 /// Sentence represented as a vector.
 ///
 /// This data type represents a sentence as vectors (`Vec`) of tokens and
 /// part-of-speech indices. Such a vector is typically the input to a
```
Comment (twuebi) [inline toponn/src/input.rs]: Embeddings, not indices, right?

File: toponn/src/input.rs [hunk 2]
```diff
@@ -34,7 +67,7 @@ impl SentVec {
 
     /// Decompose the sentence vector into vectors of token indices and
     /// part-of-speech tag indices.
```
Comment (twuebi) [inline toponn/src/input.rs]: Embeddings

File: toponn/src/tensorflow/collector.rs [hunk 3]
```diff
@@ -54,17 +54,24 @@ impl TensorCollector {
             .for_each(|(idx, labels)| batch_seq_lens[idx] = labels.len() as i32);
 
         let max_seq_len = self.cur_labels.iter().map(Vec::len).max().unwrap_or(0);
```
Comment (twuebi) [inline toponn/src/tensorflow/collector.rs]: cur_labels cannot be empty, unwrap() would suffice.

**Review by twuebi — APPROVED** [review]
Few nitpicks

**Comment by danieldk** [comment]
Thanks! Nitpicks solved.

**Pull request merged**